model/url.yaml

```diff
@@ -37,3 +37,94 @@ groups:
         type: string
         brief: 'The [URI fragment](https://www.rfc-editor.org/rfc/rfc3986#section-3.5) component'
         examples: ["SemConv"]
+      - id: registered_domain
+        requirement_level: opt_in
+        type: string
+        brief: >
+          The highest registered url domain, stripped of the subdomain.
+
+          For example, the registered domain for "foo.example.com" is "example.com".
+
+          This value can be determined precisely with a list like the public suffix
+          list (`http://publicsuffix.org`). Trying to approximate this by simply taking
+          the last two labels will not work well for TLDs such as "co.uk".
+        examples: [ "example.com" ]
+      - id: subdomain
+        requirement_level: opt_in
+        type: string
+        brief: >
+          The subdomain portion of a fully qualified domain name includes all of
+          the names except the host name under the registered_domain. In a partially
+          qualified domain, or if the the qualification level of the full name cannot
+          be determined, subdomain contains all of the names below the registered domain.
+
+          For example the subdomain portion of `www.east.mydomain.co.uk` is "east".
+          If the domain has multiple levels of subdomain, such as `sub2.sub1.example.com`,
+          the subdomain field should contain "sub2.sub1", with no trailing period.
+        examples: [ "east" ]
+      - id: top_level_domain
+        requirement_level: opt_in
+        type: string
+        brief: >
+          The effective top level domain (eTLD), also known as the domain suffix,
+          is the last part of the domain name. For example, the top level domain
+          for example.com is "com".
+
+          This value can be determined precisely with a list like the public suffix list
+          (`http://publicsuffix.org`). Trying to approximate this by simply taking the last
+          label will not work well for effective TLDs such as `co.uk`.
+        examples: [ "co.uk" ]
+      - id: username
```

Comment:
Should we add `username` and `password`? @trask [linked to them](https://github.com/open-telemetry/opentelemetry-specification/issues/3405#issuecomment-1512268209) being deprecated? https://www.ietf.org/rfc/rfc3986.txt

Reply:
Hmm, I'm was not aware of those being deprecated. 
@AlexanderWert @trisch-me do you you know more about this and if those would be deprecated from ECS?

Reply:
Instrumentations also cover older http clients which would still contain `username` and `password`.
So, I think `username` and `password` could still be useful as `opt-in` attributes.